### PR TITLE
Add new PDA install procedure

### PR DESCRIPTION
**Pull request by oTTer-Chief**

### CODE REVIEW

File: contrib/pda_inst.sh [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
+}
+
+install_lib()
+{
+  cd /tmp
+  wget https://github.com/cbm-fles/pda/releases/download/$1/libpda4_$1-1_amd64.deb
+  apt-get install ./libpda4_$1-1_amd64.deb
+}
+
+# Default install is done via Debian packages. Functions blow are kept for reference.
```
Comment (cuveland) [inline contrib/pda_inst.sh:24]:
```suggestion
# Default install is done via Debian packages. Functions below are kept for reference.
```

**Comment by cuveland** [comment]
Let's fix the typo and merge this...

**Comment by schintke** [comment]
Or should we add quotes around variable usages such as $1 -> "$1" etc, so that spaces or other weird characters in those variables cannot harm the commands?

**Comment by oTTer-Chief** [comment]
> Or should we add quotes around variable usages such as $1 -> "$1" etc, so that spaces or other weird characters in those variables cannot harm the commands?

Would certainly improve it. However, the variables are only internal to the script and so I think it is fine.

**Pull request merged**